README.md

```diff
@@ -47,152 +40,174 @@ Do you want to know how your shipment is going and how long it will arrive? You
-| :----------------------------: | :---------------------------------------------------------------------------------------------------------------------------------: |
-|        ![author-pic][]         |                                                             ![banner][]                                                             |
 
-## Contributing
+**Commits:**
 
-[![contributions welcome][contributions-welcome]][issues-url]
+Like branches, commits must also follow a format. Notice how the commit message varies if it is a feature or a bug fix.
 
-🤝 Contributions, issues and feature requests are welcome!
-Feel free to check the [issues page][issues-url].
+**feat**: added header to main page
 
-## Show your support
+**fix**: removed bad media queries in rating component
 
-🤗 Give a ⭐️ if you like this project!
+**Pull Request (PR):**
 
-- API: Powered by [Google Maps](https://developers.google.com/maps/documentation/javascript/overview)
+The PR’s are requests that as developers we make once the solution of an issue is completed and we want the code to be integrated into the master branch of the project. Before that, we must ensure that our code is fully functional and adhering to best practices.
 
-- <https://blog.logrocket.com/deep-dive-into-lefthook-react-native/>
-- <https://blog.logrocket.com/build-robust-react-app-husky-pre-commit-hooks-github-actions/>
-- Favicon from [favicon.io](https://favicon.io/emoji-favicons/)
-- Production deployment using [Netlify](https://app.netlify.com/sites/azordev-dasher-user/overview)
-- <https://www.seoptimer.com/meta-tag-generator>
-- <https://dev.to/nitzano/linting-docker-containers-2lo6?utm_source=lefthook>
+Once we have done a commit & push in our branch, a notification with a yellow background will appear in the github repository and will ask us to continue with the PR. In the TextArea to perform the PR you will find several sections that you want to complete: A brief description of what you did, a snapshot (if applicable), the tasks that were completed and the steps to view them.
 
-## License
 
-[![License][badge-apache]][apache-license]
+## FAQ:
 
-📝 This project is licensed under the [Apache 2](LICENSE)\
-Feel free to fork this project and improve it!
+**How do I create a new component?**
 
+In the component folder, create a new file with the .jsx extension. Make sure to put an appropriate name to the functionality that you are going to develop.
+
+## Collaborators
+
+| [Israel Laguan][author-github] | ![email-icon][] Email me to [contact@israellaguan.com][author-email] / ![linkedin-icon][] Connect to [my Linkedin][author-linkedin] |
+| :----------------------------: | :---------------------------------------------------------------------------------------------------------------------------------: |
+|        ![author-pic][]         |                                                             ![banner][]                                                             |
```

Comment:
Please remove banner, seems like it breaks for others, I prefer everyone looks similar instead of stands out

Reply:
It would be nice to have a single "azordev" Banner. What do you think?

Reply:
Lets include this in a new issue